purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -753,6 +756,25 @@ internal class BillingWrapper(
 
     override fun isConnected(): Boolean = billingClient?.isReady ?: false
 
+    override fun showInAppMessagesIfNeeded(activity: Activity) {
+        val inAppMessageParams = InAppMessageParams.newBuilder()
+        .addInAppMessageCategoryToShow(InAppMessageParams.InAppMessageCategoryId.TRANSACTIONAL)
+        .build()
+
+        billingClient?.showInAppMessages(activity,
+            inAppMessageParams
+        ) { inAppMessageResult ->
+            if (inAppMessageResult.responseCode == InAppMessageResult.InAppMessageResponseCode.NO_ACTION_NEEDED) {
+                // No in-app message was available.
+                // TODO: log
+            } else if (inAppMessageResult.responseCode
+                == InAppMessageResult.InAppMessageResponseCode.SUBSCRIPTION_STATUS_UPDATED
+            ) {
+                // TODO: post receipt
```

Comment:
we'd need to do a syncPurchases

Reply:
Hmm do we need to? If I remember correctly, when a purchase recovers, it would still be the same token, so it's already sent and we shouldn't need to do anything right?

Reply:
However, we probably do need to get the new customer info from the backend. I forgot that. Adding

Reply:
Added in https://github.com/RevenueCat/purchases-android/pull/1290/commits/a8eb16f5b5c433520dac67a131e41e84ecf98fc1